aten/src/ATen/core/TensorBase.h

```diff
@@ -265,6 +265,11 @@ class TORCH_API TensorBase {
     return impl_->is_contiguous(memory_format);
   }
 
+  // check Note[definitely_contiguous_fast]
+  bool definitely_contiguous_fast(at::MemoryFormat memory_format=at::MemoryFormat::Contiguous) const {
```

Comment:
Why not name this "definitely_contiguous"?
Also it would be good to have the same comment that is in the Contiguity.h definition of definitely_contiguous here.
I know methods here are not well documented but since we already have doc for this one, let's use it!

Reply:
I added the _fast suffix because we're caching for definitely_contiguous results. This means that if runtime assertions are added later that could infer the tensor is always contiguous after caching, this method will still return False. The _fast suffix is intended to refer to this specific case. I'm not strongly attached to the suffix, though - if you think it's better without it, I can remove it.

Reply:
I can add the same comment here instead of referring to the note.